one-time-login.php

```diff
@@ -17,11 +17,12 @@
  *
  * @param WP_User|null $user  ID, email address, or user login for the user.
  * @param int          $count           Generate a specified number of login tokens (default: 1).
- * @param bool         $delay_delete   Delete existing tokens after 15 minutes, instead of immediately.
+ * @param bool         $delay_delete    Delete existing tokens after 15 minutes, instead of immediately.
+ * @param int          $expiry          Delete existing token after $expiry minutes from creation, even if not used (default: 0 - not expiry).
  *
  * @return array
  */
-function one_time_login_generate_tokens( $user, $count, $delay_delete ) {
+function one_time_login_generate_tokens( $user, $count, $delay_delete, $expiry ) {
```

Comment:
```suggestion
function one_time_login_generate_tokens( $user, $count, $delay_delete, $expiry = 0 ) {
```

Should we default to `0` so we don't break existing calls to this function?

Reply:
as $delay_delete, default value is set on calling function. I can change this but I think this way keeps the current standard.

Reply:
I think it'd be best to make this change.